app/views/root/_error_page.html.erb

```diff
@@ -12,29 +12,23 @@
 <% content_for :body_classes, "mainstream error" %>
 
 <% content_for :wrapper_content do %>
-<main id="content" role="main" class="group">
-  <header class="page-header group">
-    <div>
-      <h1><%= heading %></h1>
-    </div>
-  </header>
-  <div class="article-container">
-    <article role="article" class="group">
-      <div class="inner">
+  <main id="content" role="main" class="govuk-main-wrapper govuk-main-wrapper--l">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l"><%= heading %></h1>
```

Comment:
Could a [page title](https://components.publishing.service.gov.uk/component-guide/title) component be used here?

Reply:
good shout @injms, I've updated it to use the component as suggested.